src/main/java/pvs/app/entity/Project.java

```diff
@@ -29,43 +34,7 @@
     )
     private Set<Repository> repositorySet = new HashSet<>();
 
-    public Long getProjectId() {
-        return projectId;
-    }
-
-    public void setProjectId(Long projectId) {
-        this.projectId = projectId;
-    }
-
-    public Long getMemberId() {
-        return memberId;
-    }
-
-    public void setMemberId(Long memberId) {
-        this.memberId = memberId;
-    }
-
-    public String getName() {
-        return name;
-    }
-
-    public void setName(String name) {
-        this.name = name;
-    }
-
-    public String getAvatarURL() {
-        return avatarURL;
-    }
-
-    public void setAvatarURL(String avatarURL) {
-        this.avatarURL = avatarURL;
-    }
-
-    public Set<Repository> getRepositorySet() {
-        return repositorySet;
-    }
-
-    public void setRepositorySet(Set<Repository> repositorySet) {
-        this.repositorySet = repositorySet;
-    }
+    // 0: deleted; 1: normal
+    @Column(columnDefinition = "integer default 1")
+    private Integer status;
```

Comment:
```suggestion
    @Column(name = "markAsRemoved")
    private boolean markAsRemoved;
```

Reply:
If the getter of `boolean` porperty is prefixed "is" to the property name, I think that the property name could be just "removed". When we call the getter, `isRemoved()` is also readable.

Reply:
3f92894